### ISSUE TITLE
Detonate command does not work

### ISSUE THREAD

**Issue by untuned** [Exiled-Team/CustomItems#66]
**Describe the bug**
When attempting to run `.detonate` in the in-game console, an error is produced.

**To Reproduce**
1. Give yourself C4-119
2. Place it on the ground
3. Run `.detonate` in the console
4. Observe the error

**Expected behavior**
The C4 charge detonates.

**Server logs**
There are no error logs in the server's console, but this shows in the in-game console:
![in-game console](https://untnd.me/89RLEOqvr)

**EXILED Version ("latest" is not a version):**
`Exiled.Loader:4.1.6`

**Results of `show plugins` command in console:**
https://paste.hep.gg/gymub.yaml

**Additional context**
N/A

**Comment by DinoNuggets2** [comment]
You might have the config on that requires the player to have a radio on them. Check the C4 config and make sure **require_detonator** is set to false

**Comment by untuned** [comment]
> You might have the config on that requires the player to have a radio on them. Check the C4 config and make sure **require_detonator** is set to false

I did notice this fact afterwards. Thank you!
I would like to see maybe a message saying that they don't have the required item though.

**Comment by DinoNuggets2** [comment]
Yeah, I had the same issue when I started using this plugin, that would also be cool if it told the player they need the specific item
